modules/oke/kubernetestools.tf

```diff
@@ -4,10 +4,6 @@
 data "template_file" "helm_enabled" {
   template = file("${path.module}/scripts/install_helm.template.sh")
 
-  vars = {
-    helm_version       = var.helm.helm_version
-  }
-
   count = var.oke_operator.operator_enabled == true && var.helm.helm_enabled == true ? 1 : 0
```

Comment:
No need to check if helm is enabled as discussed. So check only if operator is enabled.

Reply:
helm variables were removed